git-merge-conflict.md

```diff
@@ -0,0 +1,52 @@
+1. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and delete the recipe scrap then commit both of these changes.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together. There are a number of ways that your group can accomplish that, and you should discuss them to figure out which strategy to employ:
+
+- **Round robin**: This strategy involves direct coordination between everyone in the group, resulting in a strictly ordered merge sequence.
+  - One at a time each person pushes their changes and resolves any merge conflicts.
+  - The next person in turn pulls those changes and pushes their own until everyone's changes have been pushed.
+- **Ad-hoc**: This approach relies the most upon Git's merge coordination capabilities to ensure that everyone's changes get merged in properly.
+  - Essentially, everyone attempts to `git push` at once.
+  - The first person to get there doesn't have to do a merge, but everyone after that probably will.
+- **Branched**: This approach is kind of a mix of the other two.
+  - Within your group split off into pairs.
+  - Both people in the pair create a branch _with the same name_. You can create a branch with `git checkout -b <branch name>`.
+  - Both people in the pair push their changes to the branch (`git push origin <branch name>`).
+  - Whoever ends up pushing second will have to merge in the other person's changes.
+  - After merging the pair's changes together, merge the result into the `master` branch:
+    - `git checkout master`
+    - `git merge <branch name>`
+  - Push the `master` branch up to GitHub.
+  - The first pair to complete this won't have to do any more merging, but the other two pairs will.
+
+Once your recipe file has been completed everyone should review it for completeness. Make sure that none of the lines from your scrap are missing from the final result.
+
+If everyone agrees that the recipe looks correct one person should submit a PR to the original [carnival-recipes](https://github.com/Ada-C7/carnival-recipes) repo.
+
+### Advice
+There are a few things to look out for when working through this activity:
+
+- The end result should be a real, usable recipe. The original recipe files were all complete works from various recipe websites online.
+- If you find that the result of one of your merges is missing critical ingredients or any steps, you should go back and double check that you didn't lose any lines while resolving a merge conflict.
+- Your recipe scraps may have duplicate lines. We've added these in to more accurately portray the full range of potential merge conflicts -- you should get rid of any duplicates while merging.
```

Comment:
I feel like this section would be good to move above the detailed options in the "Building The Recipe" section. I went through the whole thing and probably would not have seen this part about the duplicate lines until it was too late